tests/epics/test_signals.py

```diff
@@ -156,24 +149,49 @@ class MyEnum(str, Enum):
     c = "Ccc"
 
 
-def integer_d(value):
-    return dict(dtype="integer", shape=[])
+_common_metadata = {
+    "units": ANY,
+    "upper_disp_limit": ANY,
+    "lower_disp_limit": ANY,
+    "upper_ctrl_limit": ANY,
+    "lower_ctrl_limit": ANY,
+}
 
+_metadata: Dict[str, Dict[str, Any]] = {
+    "string": {"timestamp": ANY},
+    "integer": _common_metadata,
+    "number": {**_common_metadata, "precision": ANY},
+    "enum": {},
+}
```

Comment:
I've tried a few arrangements of the metadata, but because it's partially inconsistent I wasn't confident with any of them. I'd like to be able to get it back to descriptor(value) == backend.get_descriptor() rather than descriptor(values).items() <= backend.get_descriptor().items()